flake.nix

```diff
@@ -73,8 +73,11 @@
             false # Allows us to easily switch on materialization checking
           , system, sourcesOverride ? { }, ... }@args: rec {
             sources = inputs // sourcesOverride;
-            allOverlays = import ./overlays args;
-            inherit config nixpkgsArgs;
+            allOverlays = import ./overlays (args // { inherit sources; });
+            inherit config;
+            nixpkgsArgs = {
+              inherit config overlays;
```

Comment:
why is this different?

Reply:
@michaelpj It pulls `overlays` from the current `rec`, which uses our overridden sources and other args.

Reply:
Worth a comment. We have so many things in this file with the same name :sweat_smile:

Reply:
Commented.